YeRoBattle.Engine/BattleCalculator.cs

```diff
@@ -31,6 +32,62 @@ public void Hit(Character attacker, Character defender)
             {
                 defender.IsDead = true;
             }
+        }
+        public void Healing(Character character, Character target)
+        {
+            var heal = character.HealPower;
+
+            if (heal > 0) // heal character 
+            {
+                _logger.WriteLine($@"character {character.Name} has healed {character.Name} by {heal}");
+                character.Health = character.Health + heal;
+            }
+            if (character.CurrentHealth == character.Health)
+            {
+                _logger.WriteLine($@"character {character.Name} has healed {character.Name} by {heal}");
+                character.HealPower = 0;
+            }
+            else
+            {
+                _logger.WriteLine($@"character {character.Name} has healed {character.Name} by {heal}");
+                character.Health = character.Health + heal;
```

Comment:
again...no need to change Heath property..look comments above

Reply:
done